src/Service.GraphQLBuilder/Queries/QueryBuilder.cs

```diff
@@ -24,6 +24,8 @@ public static class QueryBuilder
         public const string ORDER_BY_FIELD_NAME = "orderBy";
         public const string PARTITION_KEY_FIELD_NAME = "_partitionKeyValue";
         public const string ID_FIELD_NAME = "id";
+        public const int MAX_PAGE_LIMIT = 1000;
```

Comment:
Can we reference constants from single place? I see PaginationOptions also has these.
Also, isnt MAX_PAGE_LIMIT suppoesed to be 10000?

Reply:
fixed. this was old un-referenced code that i missed removing. thanks for catching it.